### ISSUE TITLE
Why dice scores mentioned in repository and paper are different?

### ISSUE THREAD

**Issue by steve-seagal** [Project-MONAI/research-contributions#90]
![image](https://user-images.githubusercontent.com/103193487/184074704-2a5428ab-12b0-4bc0-8ac7-e0328e2547a2.png)
![image](https://user-images.githubusercontent.com/103193487/184074967-b4365957-ba18-4d37-a6e9-4c31ce81d8e6.png)

Dice score is 0.918 in the paper and 0.8225/0.8186 in the repository. Why there is a huge gap in between?
Are they evaluated on validation set or test set? And what is overlap parameter of score of 0.918 in the paper?

Thank you.

**Comment by tangy5** [comment]
Thanks for the interests on the work. The repository is build as a tutorial for multi-organ segmentation using public available data and easy-to-access GPU resources such as single GPU training. The paper performance is trained using Distributed training, tuned parameters, more than 20 models ensemble, and with collaborated additional data (80 in total for training and validation). These configurations are different from single GPU, single model, 30 public available BTCV data in the tutorial. 
Thanks!
